tests/buyers/views/test_buyers.py

```diff
@@ -1969,6 +1969,14 @@ def setup_method(self, method):
                 "blah": [False, True]
             }
         ]
+        self.bad_response = {
+            "supplierName": "Tea Boy Ltd.",
+            "respondToEmailAddress": "teaboy@example.com",
+            "availability": "Tomorrow",
+            "essentialRequirementsMet": False,
```

Comment:
@idavidmcdonald Hrm,  is this desired?  The current term system lets you submit a useless response that the buyer can never access,  is there a reason for this?  Probably worth investigating? 

I am all in favour of not needing this but maybe we need to check there's not some reasoning we are not aware of for the old functionality?